glide/src/main/java/com/google/accompanist/glide/Glide.kt

```diff
@@ -66,175 +63,117 @@ object GlideImageStateDefaults {
     }
 }
 
-@Suppress("NOTHING_TO_INLINE")
 @Composable
-inline fun rememberGlidePainter(
+fun rememberGlidePainter(
     data: Any?,
     requestManager: RequestManager = GlideImageStateDefaults.defaultRequestManager(),
-    noinline shouldRefetchOnSizeChange: (currentState: ImageLoadState, size: IntSize) -> Boolean = { _, _ -> false },
-    noinline requestBuilder: (RequestBuilder<Drawable>.(size: IntSize) -> RequestBuilder<Drawable>)? = null,
+    shouldRefetchOnSizeChange: (currentState: ImageLoadState, size: IntSize) -> Boolean = { _, _ -> false },
+    requestBuilder: (RequestBuilder<Drawable>.(size: IntSize) -> RequestBuilder<Drawable>)? = null,
     fadeIn: Boolean = false,
     fadeInDurationMs: Int = 1000,
     @DrawableRes previewPlaceholder: Int = 0,
-): Painter = rememberLoadPainter(
-    state = rememberGlideImageState(
-        data = data,
-        requestManager = requestManager,
+): LoadPainter<Any> {
+    val updatedRequestBuilder by rememberUpdatedState(requestBuilder)
```

Comment:
the issue you are solving with this should probably be solved on a different level. the real issue is that when rememberLoadPainter() is reexecuted with an updated loader lambda it still continues using the old one. 
so I suggest to wrap loader param in rememberLoadPainter inside a rememberUpdatedState instead

Reply:
I went another way, and made the loader a `@Stable interface`. Feels closer to what we actually want.